README.md

```diff
@@ -1,5 +1,7 @@
 # SPV WALLET WEB FRONTEND
 
+[SPV Wallet GitBook](https://bsvblockchain.gitbook.io/docs)
```

Comment:
IMHO this link should be introduced with some text, e.g. what it is and what an user can find there.